docs/quickstart.rst

```diff
@@ -387,7 +387,7 @@ Let's install the tools we need for Security Monkey::
 Setup Postgres
 --------------
 
-*For production, you will want to use an AWS RDS Postgres database.*  For this guide, we will setup a database on the instance that was just launched.
+*For production, you will want to use an AWS RDS Postgres database (or Cloud SQL Postgres, which is currently in beta).*  For this guide, we will setup a database on the instance that was just launched.
```

Comment:
How about something like:

*For production, you will want to use your cloud provider's managed Postgres database (such as AWS RDS Postgres or Google Cloud SQL Postgres) for improved reliability.*